contracts/modules/cover/Cover.sol

```diff
@@ -210,7 +210,7 @@ contract Cover is ICover, MasterAwareV2, IStakingPoolBeacon, ReentrancyGuard, Mu
       */
     }
 
-    uint nxmPriceInCoverAsset = pool().getInternalTokenPriceInAsset(params.coverAsset);
```

Comment:
looks like this function isn't used anymore and can be removed from the Pool contract

Reply:
Can't currently add Ramm as a dep to Cover as it exceeds the max contract size, so calling it via the Pool is the current workaround.